homeassistant/components/deconz/lock.py

```diff
@@ -50,11 +52,10 @@ def async_add_lock_from_sensor(_: EventType, lock_id: str) -> None:
     )
 
 
-class DeconzLock(DeconzDevice, LockEntity):
+class DeconzLock(DeconzDevice[_LockDeviceTypeT], LockEntity):
```

Comment:
```suggestion
class DeconzLock(DeconzDevice[Union[DoorLock, Lock]], LockEntity):
```
`DeconzLock` isn't specialized further, thus the TypeVar isn't necessary here.